### ISSUE TITLE
not running the kxss

### ISSUE THREAD

**Issue by sushantdhopat19** [Emoe/kxss#2]
i put cmd kxss -h this didnt get any ouput not get any error then i run also different cmd this also not give any ouput 

echo "http://testphp.vulnweb.com/search.php?test=123" | kxss

not give any output

**Comment by Emoe** [comment]
Sorry for the late response.
It is normal that kxss -h doesn't show any help. kxss waits for input over stdIN and needs no other flags like "-h".

Your other Command should work. How did you install the tool?
I just installed the tool on a new system to test it using the following command:
`go get github.com/Emoe/kxss`

And the tool just runs fine. [Here's a Gif](https://drive.google.com/file/d/1hGEfzko16WkkmEO6PhodL5ouRYUFwlB9/view?usp=sharing), i recorded while running it.

**Comment by p0nd4hxr** [comment]
no response  kxss cmd
i installed as u said
echo "https://www.**********.***/event_register.php?event=177" | kxss

**Comment by notorious1337** [comment]
bro itsl literally not working
it is giving no output

**Comment by sushantdhopat** [comment]
It's not working / not showing any output  because sometimes site is not vulnerable try to use on vulnerable platform and see

**Comment by hcr5246** [comment]
any updates?
this is what I get (kxss: command not found
)

**Comment by 1bdool492** [comment]
kxss: command not found

**Comment by hxhBrofessor** [comment]
I don't believe it is working with the latest golang version.. go get is deprecated and you have to use  go install github.com/tomnomnom/httprobe@latest

It's not working for either...

**Comment by Nikakz** [comment]
Nao funciona

**Comment by cyb3r-n3rd** [comment]
did it work it installed it via git clone & using go install not getting output of any form passed the url as well not output

**Comment by xspatrian** [comment]
same problem

**Comment by bayhaqibnu8** [comment]
I can finally solve the problem of kxss not running properly, follow this method:
1. sudo rm -rf /usr/bin/kxss /usr/local/bin/kxss
2. git clone https://github.com/Emoe/kxss.git
3. cd kxss
4. go env -w GO111MODULE=auto
5. go build
6. sudo cp kxss /usr/bin
